src/main/java/helper/Constant.java

```diff
@@ -11,9 +11,10 @@
     public static final int TIMES_WAIT_ELEMENTS = 120;
     public static final int LONG_TIME_WAIT = 30;
     public static final String DATE_FORMAT = "M/d/yyyy";
-
+    public static final String BLANK_EMAIL = "";
+    public static final String PASSWORD = "minhvip1997";
```

Comment:
already existed

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.